renderer/components/nettests/index.js

```diff
@@ -98,8 +100,9 @@ export const testGroups = {
 export const tests = {
   // TODO: (sarathms) Replace each one with their own implementations when ready
   web_connectivity,
-  http_header_field_manipulation: web_connectivity,
-  http_invalid_request_line: web_connectivity,
+  http_header_field_manipulation,
+  http_invalid_request_line,
+  facebook_messenger: web_connectivity,
```

Comment:
Is it desired that this points to `web_connectivity`, are you doing that because you don't have an implementation of `facebook_messenger` yet so you are using web_connectity as a placeholder?

Reply:
Yes. It is a placeholder. It doesn't make a lot of difference, because that implementation isn't stable anymore anyway. This assignment was only to avoid errors at the `MeasurementContainer` level.